### DISCUSSION TITLE
de-duplication

### DISCUSSION THREAD

**Discussion by r3pairmen** [projectdiscovery/nuclei#3362] [Q&A]
hi,
i need a help folks, nuclei auto deduplicate similar hostnames and ignores http:// or https:// protocol.

such as http://127.0.0.1
             https://127.0.0.1    is de duplicated in one host for scanning, i need to disable this and scan as it is from the file.

any idea
thanks

**Comment by ehsandeep** [comment]
@r3pairmen I'm not able to reproduce this behavior, do you example to share?

```console
$ cat target.txt

http://127.0.0.1
https://127.0.0.1
```

```console
$ nuclei -id tech-detect -list target.txt 

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.8.9

		projectdiscovery.io

[INF] Using Nuclei Engine 2.8.9 (latest)
[INF] Using Nuclei Templates 9.3.7 (latest)
[INF] Templates added in last update: 58
[INF] Templates loaded for scan: 1
[INF] Targets loaded for scan: 2
[INF] No results found. Better luck next time!
```

**Comment by r3pairmen** [comment]
hi,
i for some reason cannot share my list of ip address with pre-pended http:// and https:// proto- but like i mentioned above my list contains around 150 domains in http:// and same domain in https:// 

unless i misunderstood something-please take a look at the screenshot.
might help you

<img width="555" alt="Screenshot_5" src="https://user-images.githubusercontent.com/89855073/221402444-1611c705-63f5-43bb-b778-3e8a6951cf28.png">
